pkg/operator/sync_all_v4_00.go

```diff
@@ -1,21 +1,22 @@
 package operator
 
 import (
+	"github.com/openshift/library-go/pkg/operator/events"
 	scsv1 "github.com/openshift/service-ca-operator/pkg/apis/serviceca/v1"
 )
 
 // sync_v4_00_to_latest takes care of synchronizing (not upgrading) the thing we're managing.
 // most of the time the sync method will be good for a large span of minor versions
-func sync_v4_00_to_latest(c serviceCertSignerOperator, operatorConfig *scsv1.ServiceCA) error {
-	err := syncSigningController_v4_00_to_latest(c, operatorConfig)
+func sync_v4_00_to_latest(c serviceCAOperator, eventRecorder events.Recorder, operatorConfig *scsv1.ServiceCA) error {
```

Comment:
Why add the `eventRecorder` when you're passing the whole `serviceCAOperator` struct which has it included?

Reply:
🤦‍♀️ removed